sandbox/chris-gan/implicit_style_fixed_samples/discriminator.py

```diff
@@ -0,0 +1,129 @@
+#!/bin/python3
+
+#!/bin/python
+
+import numpy as np
+import torch
+import torch.nn as nn
+import torch.nn.functional as F
+
+
+class ResidualBlock(nn.Module):
```

Comment:
It looks like the `ResidualBlock` class is duplicated across `discriminator.py` and `nnlib.py`. Is there a reason for that?